baselines/powerplatform.md

```diff
@@ -4,42 +4,40 @@ The Microsoft Power Platform is a group of applications involving
 low-code application development, business intelligence, a custom chat
 bot creator, and app connectivity software. The following summarizes the
 Power Platform applications and other applications frequently used by
-Power Platform applications.
+Power Platform.
 
```

Comment:
Please update the title as per new style guide. (viz. CISA M365 Security Configuration Baseline for Power Platform)